src/galaxy/CustomSystem.cpp

```diff
@@ -638,7 +626,7 @@ static void RegisterCustomSystemsAPI(lua_State *L)
 	register_class(L, LuaCustomSystemBody_TypeName, LuaCustomSystemBody_meta);
 }
 
-void CustomSystemsDatabase::Init()
+void CustomSystemsDatabase::Load()
```

Comment:
Init was the convention for this kind of class, you don't need to know what it's doing inside just that things are being initialised. Changing it to Load is currently correct for this method but doesn't declare the intent of it, nor conforms to the others that are like it.

Reply:
I would highlight it _load_ files, not only initialize some value...
Don't know: I can restore it or (rather?) call it 'InitAndLoad'.

Reply:
Nah keep it ,I'm just wondering as to your motivation behind the change, since it doesn't seem necessary or related to the rest of the PR... just something you did that got caught up in it.